ruby/lib/minitest/queue/runner.rb

```diff
@@ -134,7 +135,10 @@ def report_command
       def run_tests_in_fork(queue)
         child_pid = fork do
           Minitest.queue = queue
-          Minitest::Reporters.use!([Minitest::Reporters::BisectReporter.new])
+          Minitest::Reporters.use!([
+            Minitest::Reporters::BisectReporter.new,
+            Minitest::Reporters::StatsdReporter.new,
```

Comment:
I doubt you want the reporter here. This is for the bisect flow.

Reply:
Where do I register it instead?

Reply:
Here I think: https://github.com/Shopify/ci-queue/blob/37c9de8ad2e96eb6f43b1538c2c182ba798a0de0/ruby/lib/minitest/queue/runner.rb#L46-L50